include/sparrow/arrow_array_schema_proxy.hpp

```diff
@@ -374,14 +377,68 @@ namespace sparrow
          */
         [[nodiscard]] SPARROW_API arrow_proxy view() const;
 
+        /**
+         * Check if the proxy owns the `ArrowArray` and not only a pointer to it.
+         */
         [[nodiscard]] SPARROW_API bool owns_array() const;
+
+        /**
+         * Extract the `ArrowArray` from the proxy. The `ArrowArray` is not released when the proxy is
+         * destroyed.
```

Comment:
I would insist a little bit more on the need fo rthe user to release the array after use: "Extracts the `ArrowArray` from the proxy, and transfers the responsibility to release it after usage to the caller.

Reply:
ok